airflow/providers/amazon/README.md

```diff
@@ -95,7 +95,7 @@ pip install apache-airflow-backport-providers-amazon[apache.hive]
 | [apache-airflow-backport-providers-postgres](https://github.com/apache/airflow/tree/master/airflow/providers/postgres)       | postgres    |
 | [apache-airflow-backport-providers-ssh](https://github.com/apache/airflow/tree/master/airflow/providers/ssh)                 | ssh         |
 
-# Provider class summary
+# Provider classes summary
 
 All classes in Airflow 2.0 are in `airflow.providers.amazon` package.
```

Comment:
```suggestion
All operator classes in Airflow 2.0 are in `airflow.providers.amazon` package.
```
WDYT?

Reply:
Yep. I updated the description and also reviewed and cleaned up documentation for the backport packages in README.md to explain a bit better what the packages are for and how to install them and about the watchout for installing the backport packages in the same namespace as 'apache-airflow'.